app/src/main/java/com/infomaniak/mail/data/models/Thread.kt

```diff
@@ -192,6 +214,15 @@ class Thread : RealmObject {
         FOLDER(R.string.searchFilterFolder),
     }
 
+    @Serializable
+    data class ThreadResult(
+        val threads: List<Thread>? = null,
+        @SerialName("resource_previous")
+        val resourcePrevious: String?,
+        @SerialName("resource_next")
+        val resourceNext: String?,
+    )
```

Comment:
For coherence with the other models of the project, put this in a separateFile along with Thread in a Thread package

Reply:
🤔 yes we can do it